wpilibc/src/main/native/include/frc/simulation/DifferentialDrivetrainSim.h

```diff
@@ -119,6 +119,20 @@ class DifferentialDrivetrainSim {
    */
   units::ampere_t GetCurrentDraw() const;
 
+  /**
+   * Resets the system state.
+   *
+   * @param state The state to reset to.
+   */
+  void ResetState(const Eigen::Matrix<double, 7, 1>& state);
+
+  /**
+   * Resets the system pose.
+   *
+   * @param pose The pose to reset to.
+   */
+  void ResetPose(const frc::Pose2d& pose);
```

Comment:
Why ResetPose instead of simply SetPose?

Reply:
Good point.